packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -364,6 +364,11 @@ class Auth {
         })
     }
 
+    clearUserAuth() {
+        this.logger.info('Login was invalidated. Clearing login state.')
+        this.clearStorage()
+    }
```

Comment:
The intent with this function is to clear storage without re-triggering auth. However, the app still seems re-trigger auth when the state of the useAccessToken hook is updated.

Reply:
It looks like this method isn't adding new functionalities, it feels like an unnecessary new method.

Is the reason you created because `this.clearStorage()` is a private method? Well, the reason it is private is because it is dangerous to be called from outside the class. There are side effects when calling this, we never want to call it from outside of the `Auth` class to ensure state is always properly managed.

I see that you've introduced `updateCustomerPassword` method that automatically logs the user in, which seems the right approach.

My question is, under what scenario do you want to clear storage without re-triggering auth?

Reply:
Fair point. The intent of this function was primarily to provide a way to stop the auto-guest login after the password change. But since that is handled via the new `updateCustomerPassword` method now, this is no longer needed.

On other browser sessions, I agree we should re-login as guest when the auth state is reset.

Reply:
`Auth.clearUserAuth` has been removed and places that called this before are now calling `Auth.logout`